seacrowd/sea_datasets/indoner_tourism/indoner_tourism.py

```diff
@@ -0,0 +1,196 @@
+"""
+import os
+import pandas as pd
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@article{JLK,
+    author = {Ahmad Hidayatullah and Muhammad Fakhri Despawida Aulia Putra and Adityo Permana Wibowo and Kartika Rizqi Nastiti},
+    title = { Named Entity Recognition on Tourist Destinations Reviews in the Indonesian Language},
+    journal = {Jurnal Linguistik Komputasional},
+    volume = {6},
+    number = {1},
+    year = {2023},
+    keywords = {},
+    abstract = {To find information about tourist destinations, tourists usually search the reviews about the destinations they want to visit. However, many studies made it hard for them to see the desired information. Named Entity Recognition (NER) is one of the techniques to detect entities in a text. The objective of this research was to make a NER model using BiLSTM to detect and evaluate entities on tourism destination reviews. This research used 2010 reviews of several tourism destinations in Indonesia and chunked them into 116.564 tokens of words. Those tokens were labeled according to their categories: the name of the tourism destination, locations, and facilities. If the tokens could not be classified according to the existing categories, the tokens would be labeled as O (outside). The model has been tested and gives 94,3% as the maximum average of F1-Score.},
+    issn = {2621-9336}, pages = {30--35},   doi = {10.26418/jlk.v6i1.89},
+    url = {https://inacl.id/journal/index.php/jlk/article/view/89}
+}
+"""
+
+_DATASETNAME = "indoner_tourism"
+
+_DESCRIPTION = """\
+This dataset is designed for named entity recognition (NER) tasks in the Bahasa Indonesia tourism domain. It contains labeled sequences of named entities, including locations, facilities, and tourism-related entities. The dataset is annotated with the following entity types:
+
+    O (0)    : Non-entity or other words not falling into the specified categories.
+    B-WIS (1): Beginning of a tourism-related entity.
+    I-WIS (2): Continuation of a tourism-related entity.
+    B-LOC (3): Beginning of a location entity.
+    I-LOC (4): Continuation of a location entity.
+    B-FAS (5): Beginning of a facility entity.
+    I-FAS (6): Continuation of a facility entity.
+"""
+
+_HOMEPAGE = "https://github.com/fathanick/IndoNER-Tourism/tree/main"
+
+_LANGUAGES = ['ind']  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.AFL_3_0.value 
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://raw.githubusercontent.com/fathanick/IndoNER-Tourism/main/ner_data.tsv",
+}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class IndoNERTourism(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class IndoNERTourismDataset(datasets.GeneratorBasedBuilder):
```

Reply:
Implemented!

Reply:
I have an issue with performing the make command :(, I'll try to find a solution to this, but, all I have attempted failed (even downloading wsl).